src/components/styles.js

```diff
@@ -0,0 +1,39 @@
+import styled from 'styled-components';
```

Comment:
your separation of concerns is great one thing i might do different here is having these styles along with the components that use them. Components should be completely self contained, that way if you want to use it somewhere else you can just take the file the component is in as is and drop it in another project and it will work.